src/main/java/project/anemonebot/anemone/Greeting.java

```diff
@@ -1,5 +1,9 @@
 package project.anemonebot.anemone;
 
+/**
```

Comment:
Något onödig Java dokumentation om vi ska radera detta sen? 
Kör på //TODO: Remove class once this test is complete.